### DISCUSSION TITLE
app::event_clicks() design

### DISCUSSION THREAD

**Discussion by wyhinton** [fltk-rs/fltk-rs#781] [General]
I the docs, the description for ```app::event_clicks()``` we see:

> pub fn event_clicks() -> bool
> Returns the number of clicks

But doesn't the function return ```false``` for  1 click and ```true``` for 2 clicks? I think to say that it "returns the number of clicks" might lead people to think it returns an ```i32 = 1``` or an ```i32 = 2```. 

I can't say the idea of adding another Enum, ```MouseClicks::Single```/ ```MouseClicks::Double``` sounds great, but as it stands I think the docs are a little confusing.

**Comment by MoAlyousef** [comment]
You're right. Fl::event_clicks returns the number of clicks (I missed that initially). Trying it out, it can even return more than 2 clicks, adding an overload event_clicks_num() which actually returns the number of clicks seems more appropriate. I've also updated the docs.
